### DISCUSSION TITLE
Help installing into laptop

### DISCUSSION THREAD

**Discussion by baaoh** [4IceG/RM520N-GL#6] [Q&A]
Hi! I'm very new to this, and so far I thought I understand things, but now with this modem, I do NOT understand anything.

I have a new laptop, was advertised as 5G compatible, but came without a 5G module. It has antennae leads to where the module should be, has an m.2 slot, everything seems perfect.

I chose and bought the RM520N-GL, and installed it. Don't know where does which antenna go but that's for later - used the service manual to "assume" the antennae placements.

Anyway, now I've got the module all sitting snug, but windows doesn't recognize the device, and it doesn't show up in device manager. Am I assuming something horribly wrong? Is this setup NOT supposed to work at all?
My laptop model Asus Expertbook B7(B7402BFA) is supposed to run 5G out of the box but maybe my unit was discounted because it does not have the module installed?

Maybe I need to config the modem somehow before installing it in windows?

I'm very confused and there is NO information about this anywhere

Thank you very much

**Comment by 4IceG** [comment]
Hi,

It is obvious that a laptop without a modem should be significantly
cheaper (5G modems are still expensive at the moment).

The modem without antennas connected should be detected in the laptop.
Check if the modem is visible in the BIOS, maybe the modem option
requires switching something in the BIOS?

Does Windows see any unrecognized devices?

I don't know what the specification is of the m.2 port on this laptop,
whether the modem should work as USB or PCI-E. The modem should be
visible somewhere (Windows/Bios).

Does the ASUS website provide software for operating modems/modem connections?

**Reply by baaoh** [reply]
Thank you for your reply.

The SKUs that ship with the module have the Fibocom FM350 GL, maybe that's the only one supported? Asus site also has drivers for that one.
I ordered it separately, so now waiting for it to arrive.

Device manager doesn't show any USB nor Network device, even among hidden items.

BIOS has 5G options enabled, all USB options also enabled. Tried HWinfo but also no mention of the quectel.

I'm considering returning everything and purchasing the more expensive model, hoping it has the 5G module installed

**Comment by 4IceG** [comment]
I'm not an expert when it comes to laptop modems. That's why I don't know if I'm thinking correctly.

If the Fibocom FM350 GL modem was attached to the laptop, the laptop should support PCIe compatible modems.

The RM520N-GL modem can work in this mode, but it must be switched to this mode using the at commands.

You would have to buy the cheapest adapter "NGFF M.2 Key-B Slot USB 3.0 Adapter Converter Card with Standard SIM Slot For 5G 4G LTE GSM Module Modem" / "NGFF M.2 Key-B to USB 3.0" and using the adapter and at commands switch modem.
This is probably why the modem is not visible (it is set to work in USB mode by default).

There are two versions of the RM520N-GL modem available for sale, one supporting only PCIe and the other supporting both USB or PCIe mode (I assume you have this version).

P.S.
Modem may appear in Windows as an unrecognized device. So don't look for anything with "quectel" in the name. Unless I'm wrong and you have Quectel drivers installed.
